### PR TITLE
Add `TeleconsultStatusSheet`

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/1623/add-teleconsultstatussheet

### CODE REVIEW

**Comment by msasikanth** [comment]
Merge this in once #2033 is merged in

File: app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt:8]: Should we add an analytics name for this as well since it's triggered via a UI event?

File: app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusSheet.kt
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusSheet.kt:57]: Can we rename the property to reflect that it is the ID of the teleconsult radio buttons instead of just `id`?

File: app/src/main/res/values/styles_v2.xml
Comment (vinaysshenoy) [inline app/src/main/res/values/styles_v2.xml:652]: This change will apply to all buttons where this style is being used. That is expected, right?

**Review by vinaysshenoy — CHANGES_REQUESTED** [review]
Overall PR looks good. Left a couple of minor comments.

File: app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt:8]: My only concern was what if the user keep changes that radio button in the same session we will have few analytics events for that. If that's fine I can implement it.

File: app/src/main/res/values/styles_v2.xml
Comment (msasikanth) [reply app/src/main/res/values/styles_v2.xml:652]: Yes, that's expected behaviour. When a button is disabled we want it to be grey and not just keep the actual button color which will confuse the users right?

File: app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt
Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt:8]: I'd say that was the desired behaviour since it's user analytics.

File: app/src/main/res/values/styles_v2.xml
Comment (vinaysshenoy) [reply app/src/main/res/values/styles_v2.xml:652]: Yup, just wanted to confirm.

**Review by vinaysshenoy — CHANGES_REQUESTED** [review]
PR looks good, let's just add the analytics event in and we'll be good to go.

**Review by vinaysshenoy — APPROVED** [review]

**Pull request merged**
